src/scripts/main.js

```diff
@@ -1,3 +1,82 @@
 'use strict';
 
-// write code here
+const appendRow = document.querySelector('.append-row');
+const removeRow = document.querySelector('.remove-row');
+const appendColumn = document.querySelector('.append-column');
+const removeColumn = document.querySelector('.remove-column');
+
+let tableRow = document.querySelectorAll('tr');
+
+const maxCount = 10;
+const minCount = 2;
```

Comment:
usually we name such variables as constants and write in uppercase
```suggestion
const MAX_COUNT = 10;
const MIN_COUNT = 2;
```